dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/imports/bundle/persister/AbstractTrackerPersister.java

```diff
@@ -127,8 +129,13 @@ public TrackerTypeReport persist(EntityManager entityManager, TrackerBundle bund
         //
         if (isNew(bundle, trackerDto)) {
           entityManager.persist(convertedDto);
-          updateDataValues(
-              entityManager, bundle.getPreheat(), trackerDto, convertedDto, bundle.getUser());
+          handleEventValueChanges(
```

Comment:
Why is it not called `updateDataValues` anymore? is more concrete than `handle...`

Reply:
because now I'm logging changes of event properties too, so it does a bit more than just updating data values
this one calls a method that is already named `handleDataValues`, so at least there's consistency in the name

Reply:
That's an implementation detail to me.

Reply:
ok, reverted to `updateDataValues`